examples/tv-app/android/include/content-launcher/AppContentLauncherManager.cpp

```diff
@@ -73,18 +74,38 @@ void AppContentLauncherManager::HandleLaunchUrl(CommandResponseHelper<LaunchResp
     string displayStringString(displayString.data(), displayString.size());
 
     // TODO: Insert code here
-
-    const char * resStr = mCommandDelegate.sendCommand(mEndpointId, contentUrlString);
-
     LaunchResponseType response;
-    response.data   = chip::MakeOptional(CharSpan::fromCharString(resStr));
+    response.data   = chip::MakeOptional(CharSpan::fromCharString("Success"));
     response.status = ContentLauncher::ContentLaunchStatusEnum::kSuccess;
     helper.Success(response);
 }
 
 CHIP_ERROR AppContentLauncherManager::HandleGetAcceptHeaderList(AttributeValueEncoder & aEncoder)
 {
     ChipLogProgress(Zcl, "AppContentLauncherManager::HandleGetAcceptHeaderList");
+    chip::app::ConcreteReadAttributePath aPath(mEndpointId, chip::app::Clusters::ContentLauncher::Id,
+                                               chip::app::Clusters::ContentLauncher::Attributes::AcceptHeader::Id);
+    const char * resStr = mAttributeDelegate->Read(aPath);
+    ChipLogProgress(Zcl, "AppContentLauncherManager::HandleGetSupportedStreamingProtocols response %s", resStr);
+
+    if (resStr != nullptr && *resStr != 0)
+    {
+        Json::Reader reader;
+        Json::Value value;
+        reader.parse(resStr, value);
+        const char * attrId = to_string(chip::app::Clusters::ContentLauncher::Attributes::AcceptHeader::Id).c_str();
```

Comment:
what is the lifetime for to_string? 

It seems to me that we create a temporary with to_string, grabs its c_str ... but after that the temporary is deleted. Is that true? if so you need to store the std::string to ensure c_str is still valid.

Reply:
Created a local variable.